streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -368,6 +370,7 @@ public void setUncaughtExceptionHandler(final Thread.UncaughtExceptionHandler eh
      * @param queryableStoreType    accept only stores that are accepted by {@link QueryableStoreType#accepts(StateStore)}
      * @param <T>                   return type
      * @return  A facade wrapping the {@link org.apache.kafka.streams.processor.StateStore} instances
+     * @throws org.apache.kafka.streams.errors.InvalidStateStoreException if the streams are (re-)initializing
```

Comment:
And then for this `store` function, we can remove the possible thrown `InvalidStateStoreException` as well.